addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.skinshortcuts" name="Skin Shortcuts" version="1.0.12" provider-name="BigNoid,marcelveldt">
+<addon id="script.skinshortcuts" name="Skin Shortcuts" version="1.0.13" provider-name="BigNoid,marcelveldt">
```

Comment:
Can you please bump to .14, as .13 will be for the fix I introduced in .12 and will need to be requested for the repo sooner than later :)